balancer-js/package.json

```diff
@@ -28,7 +28,7 @@
         "lint:fix": "eslint ./src --ext .ts --max-warnings 0 --fix",
         "subgraph:generate": "graphql-codegen --config src/modules/subgraph/codegen.yml -r dotenv/config",
         "examples:run": "TS_NODE_PROJECT='tsconfig.testing.json' ts-node -r tsconfig-paths/register",
-        "node": "npx hardhat node --fork $(grep ALCHEMY_URL .env | cut -d '=' -f2) --fork-block-number 14828550"
+        "node": "npx hardhat node --fork $(grep ALCHEMY_URL .env | tail -1 | cut -d '=' -f2) --fork-block-number $(grep FORK_BLOCK_NUMBER .env | tail -1 | cut -d '=' -f2)"
```

Comment:
Lets add a bit of info about FORK_BLOCK_NUMBER in the README

Reply:
I added a step on hardhat configuration for setting env vars. Let me know if that's what you're expecting.

Reply:
Actually I'd rather not have that FORK_BLOCK_NUMBER required to run the node, but instead as an optional argument to override hardhat_reset on tests. I pushed changes according to this suggestion. Let me know what you think.
Ps: since this was included by @gmbronco, it would be nice to have his input as well.